src/main/java/core/basesyntax/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionService;
+import core.basesyntax.strategy.OperationHandlerStrategy;
+import java.util.List;
+
+public class TransactionServiceImpl implements TransactionService {
+    private final OperationHandlerStrategy operationHandlerStrategy;
+
+    public TransactionServiceImpl(OperationHandlerStrategy operationHandlerStrategy) {
+        this.operationHandlerStrategy = operationHandlerStrategy;
+    }
+
+    @Override
+    public void addTransferToStorage(List<FruitTransaction> transactions) {
+        FruitTransaction transaction;
+        for (FruitTransaction fruitTransaction : transactions) {
+            transaction = fruitTransaction;
```

Comment:
why do you need this local variable? can you use ```fruitTransaction``` directly?

Reply:
I forgot to delete it, it was needed while writing the code, when I was messing around with different methods.